modules/claims_api/lib/bgs_service/corporate_update_web_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class CorporateUpdateWebService < ClaimsApi::LocalBGS
+    def bean_name
+      'CorporateUpdateServiceBean'
+    end
+
+    def update_poa_access(participant_id:, poa_code:, allow_poa_access: 'y', allow_poa_c_add: 'y')
```

Comment:
If the call reports ['No Data' when `allow_poa_c_add` is lowercase](https://github.com/department-of-veterans-affairs/vets-api/blob/6f1260181a5cf09951c7df87f8393a3c2bd89a9a/modules/claims_api/app/sidekiq/claims_api/poa_vbms_updater.rb#L51), should we make the default uppercase `Y`?

Reply:
Good idea, updated in latest push.